index.html

```diff
@@ -217,6 +217,37 @@ <h6>LET'S CONNECT</h6>
             </li>
         </ul>
     </section>
+    <footer class="contact-form">
+        <h2>contact me</h2>
+        <div class="par">
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+          <ul>
+            <li class="inp">
+              <input class="txt"
+              type="text"
+              id="name"
+              name="user_name" placeholder="samuel"
+              required
+              minlength="1"
```

Comment:
- Please, kindly remove the min length, since your max length is 30 so there's no need to add the min length.